src/templates/homepage.html

```diff
@@ -8,6 +9,29 @@
   <p class="desktop-promo-separator-text">{{ _('More to Explore') }}</p>
 {% endif %}
 
+{% defer (url=anonApi('search')|urlparams(limit=11, doc_type='website'), pluck='objects', id='feed-websites') %}
+  {% if this %}
+    <div class="feed feed-items feed-home c" data-columns>
```

Comment:
Not sure. That part is temporary anyway, this block will move into [here](https://github.com/mozilla/fireplace/pull/1419/files#diff-bc2f5300a0a4ef55490cf132ebe8eaa6L12) once the feed API endpoint returns these sites, too.